src/styles/block/card.scss

```
@@ -0,0 +1,101 @@
+$custom-color: #060b35;
+
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
```

don't forget to create variables for all colors
```suggestion
  background: $c-white;
```